macros.cmake

```diff
@@ -139,7 +139,16 @@ macro(rlcAddBenchmark target)
 	ADD_EXECUTABLE(${target}Benchmark src/${target}Benchmark.cpp) 
 	ADD_EXECUTABLE(rlc::${target}Benchmark ALIAS ${target}Benchmark)
 
-	TARGET_LINK_LIBRARIES(${target}Benchmark PRIVATE benchmark::benchmark ${ARGN})
+	include(FetchContent)
+	FetchContent_Declare(
+  		abseil
+  		GIT_REPOSITORY https://github.com/abseil/abseil-cpp.git
+  		GIT_TAG        20240722.0
+	)
+	FetchContent_MakeAvailable(abseil)
```

Comment:
I am fetching the Google abseil library in this way. I am not sure if there is a better way to do the same thing.